scipy/fft/_basic.py

```diff
@@ -194,8 +198,8 @@ def ifft(x, n=None, axis=-1, norm=None, overwrite_x=False, workers=None, *,
     axis : int, optional
         Axis over which to compute the inverse DFT. If not given, the last
         axis is used.
-    norm : {None, "ortho"}, optional
-        Normalization mode (see `fft`). Default is None.
+    norm : {"backward", "ortho", "forward"}, optional
+       Normalization mode (see `fft`). Default is "backward".
```

Comment:
```suggestion
        Normalization mode (see `fft`). Default is "backward".
```

restore 4-space indentation